pkg/api/script.go

```diff
@@ -12,6 +12,17 @@ const (
 	SaveArtifacts Script = "save-artifacts"
 	// Usage i the name of the script responsible for printing the builder image's short info.
 	Usage Script = "usage"
+
+	// UserScripts is the location of scripts downloaded from user provided URL (-s flag).
+	UserScripts = "downloads/scripts"
+	// DefaultScripts is the location of scripts downloaded from default location (STI_SCRIPTS_URL environment variable).
+	DefaultScripts = "downloads/defaultScripts"
+	// SourceScripts is the location of scripts downloaded with application sources.
+	SourceScripts = "upload/src/.sti/bin"
+	// UploadScripts is the location of scripts that will be downloaded to the image during STI build.
+	UploadScripts = "upload/scripts"
+	// Source is the location of application sources.
+	Source string = "upload/src"
```

Comment:
Also, it may make it clearer in the rest of the code to suffix these with 'Dir' or something like that so you know what the constant is referring to.